tests/tcIoc_motion.py

```diff
@@ -140,12 +144,23 @@ def test_WHEN_limits_hit_THEN_motor_reports_limits(self, _, motor_pv_suffix, pv_
     def test_WHEN_axis_set_up_THEN_retries_are_not_allowed(self):
         self.ca.assert_that_pv_is(MOTOR_RTRY, 0)
 
-    def test_WHEN_enabled_set_on_plc_THEN_motor_able_is_set_correctly(self):
+    def test_WHEN_enabled_set_on_plc_THEN_motor_status_on_is_set_correctly(self):
         axis_num = 1
         self.ca.set_pv_value(ENABLE.format(axis_num), 0)
-        self.ca.assert_that_pv_is(MOTOR_ABLE_BASE.format(axis_num), 'Disable') # values are flipped vs PLC, so _able == 1 is PLC disabled etc
+        self.ca.assert_that_pv_is(MOTOR_ON_STATUS.format(axis_num), 0)
+
         self.ca.set_pv_value(ENABLE.format(axis_num), 1)
-        self.ca.assert_that_pv_is(MOTOR_ABLE_BASE.format(axis_num), 'Enable')
+        self.ca.assert_that_pv_is(MOTOR_ON_STATUS.format(axis_num), 1)
+
+    def test_WHEN_enabled_set_via_mtr_rec_THEN_PLC_is_updated(self):
+        axis_num = 1
+        self.ca.set_pv_value(MOTOR_ON_CMD.format(axis_num), 0)
+        self.ca.assert_that_pv_is(MOTOR_ON_STATUS.format(axis_num), 0)
+        self.ca.assert_that_pv_is(ENABLED.format(axis_num), 0)
+
+        self.ca.set_pv_value(MOTOR_ON_CMD.format(axis_num), 1)
+        self.ca.assert_that_pv_is(MOTOR_ON_STATUS.format(axis_num), 1)
+        self.ca.assert_that_pv_is(ENABLED.format(axis_num), 1)
```

Comment:
Same as previous comment in that I believe this can be split into two tests using Parameterised if you also agree with this?